### PR TITLE
Add replaceRoute

### PR DESCRIPTION
**Pull request by agarrharr**
I couldn't get it to build, so I just changed `build/director.js`. Obviously I couldn't change `build/director.min.js`.

### CODE REVIEW

**Comment by agarrharr** [comment]
@iammerrick @statianzo

**Comment by iammerrick** [comment]
Happy to merge once @statianzo reviews :-)

File: lib/director/browser.js [hunk 1]
```diff
@@ -220,8 +224,20 @@ Router.prototype.setRoute = function (i, v, val) {
   else {
     url = [i];
   }
+  return url;
+};
+
+Router.prototype.setRoute = function (i, v, val) {
+  var url = this.getUrl(i, v, val);
+
+  listener.setHash(url.join('/'), {replace: false});
```
Comment (statianzo) [inline lib/director/browser.js:233]: Much better. I know what `false` means now.

**Review by statianzo — APPROVED** [review]
💵

**Pull request merged**